app/routes/events/components/Admin.js

```diff
@@ -1,64 +1,64 @@
 // @flow
 
-import React, { Component } from 'react';
+import React from 'react';
 import { Link } from 'react-router';
 import type { ID, Event, ActionGrant } from 'app/models';
+import { ConfirmModalWithParent } from 'app/components/Modal/ConfirmModal';
 
 type Props = {
-  deleteEvent: (eventId: ID) => mixed,
+  deleteEvent?: (eventId: ID) => Promise<*>,
   event: Event,
   actionGrant: ActionGrant
 };
 
-type State = {
-  verifyDelete: boolean
-};
-
-export default class Admin extends Component<Props, State> {
-  state = {
-    verifyDelete: false
-  };
+const DeleteButton = ({
+  deleteEvent,
+  eventId
+}: {
+  deleteEvent: (eventId: ID) => Promise<*>,
+  eventId: ID
+}) => (
+  <li>
+    <ConfirmModalWithParent
+      title="Slett arrangement"
+      message="Er du sikker på at du vil slette dette arrangementet?"
+      onConfirm={() => deleteEvent(eventId)}
+    >
+      <a>Slett</a>
```

Comment:
`a`s should only be used with `href`s - could we style the confirm component like a link instead?

Reply:
👍